AdminUi/src/AdminUi.Infrastructure/DTOs/RelationshipDTO.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Backbone.DevelopmentKit.Identity.ValueObjects;
+
+namespace Backbone.AdminUi.Infrastructure.DTOs;
+public class RelationshipDTO
+{
+    public string Peer { get; set; }
+    public string RequestedBy { get; set; }
+    public string? TemplateId { get; set; }
```

Comment:
Hmm, I wasn't aware that this column/property is actually nullable. Can you make the property not nullable and create a migration please?

Reply:
By making this column/property not nullable, are you refering just to the property on the DTO or also the property on the actual Relationship entity?

Reply:
Since I said "and create a migration", I mean the entity of course. :)